pandas/io/formats/printing.py

```diff
@@ -382,7 +382,11 @@ def best_len(values: List[str]) -> int:
         summary = f"[{first}, {last}]{close}"
     else:
 
-        if n > max_seq_items:
+        if max_seq_items == 1:
```

Comment:
can you also cover in a test the max_seq_items == 0 case (I think we have this but maybe not). Also pls confirm that we have a test for n == max_seq_items.

Reply:
Looks like we don't have a test for `n == max_seq_items`. Will cover these in my tests. 

Just wanted to confirm that this is the relevant location for adding these?
https://github.com/pandas-dev/pandas/blob/e2dec8d03020eb2b31d376a7fcd7348977c6a319/pandas/tests/indexes/multi/test_formats.py#L92-L119

Reply:
yeah there plus some generic Index ones (look in pandas/tests/indexes/); you may have to grep around a bit

e.g.

```
~/pandas-dev$ grep -r max_seq_items pandas/tests/
pandas/tests/io/formats/test_format.py:        with option_context("display.max_seq_items", 2000):
pandas/tests/io/formats/test_format.py:        with option_context("display.max_seq_items", 5):
pandas/tests/groupby/test_groupby.py:    "max_seq_items, expected",
pandas/tests/groupby/test_groupby.py:def test_groups_repr_truncates(max_seq_items, expected):
pandas/tests/groupby/test_groupby.py:    with pd.option_context("display.max_seq_items", max_seq_items):
pandas/tests/indexes/multi/test_formats.py:    with pd.option_context("display.max_seq_items", None):
pandas/tests/indexes/multi/test_formats.py:        with pd.option_context("display.max_seq_items", 5):
pandas/tests/indexes/common.py:        with pd.option_context("display.max_seq_items", None):
pandas/tests/indexes/base_class/test_formats.py:        with cf.option_context("display.max_seq_items", 10):
```

i think `pandas/tests/indexes/base_class/test_formats.py:        with cf.option_context("display.max_seq_items", 10):`
has a number of tests as well